package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maas-schemas",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
As this is about new feature it should rather be `6.1.0` (minors are for bug fixes mainly)

Reply:
Oh, thanks! Noted!